### PR TITLE
Add a UI for visualizing the merge queue

### PR DESCRIPTION
**Pull request by casperisfine**
Followup to: https://github.com/Shopify/shipit-engine/pull/653

capture d ecran 2017-02-15 a 14 46 57

It's read only for now. I'll add controls to request / cancel a merge in a followup.

@Shopify/pipeline for review please.

### CODE REVIEW

File: app/models/shipit/pull_request.rb [hunk 1]
```diff
@@ -36,6 +38,8 @@ def required_statuses
     belongs_to :merge_requested_by, class_name: 'Shipit::User'
     has_one :merge_commit, class_name: 'Shipit::Commit'
 
+    deferred_touch stack: :updated_at
```
Comment (wvanbergen) [inline app/models/shipit/pull_request.rb]: What is this?

Comment (casperisfine) [inline app/models/shipit/pull_request.rb]: We use `stack.updated_at` to populate the Stack#show `Last-Modified` header. Anything that might change the content of that page (many things), must touch the stack to clear the cache.

`deferred_touch` is like standard `touch :stack` (akak touch parent), but it's debounced.

**Comment by gmalette** [comment]
@casperisfine I know I said this multiple times, but I must say it again:

You are simply the best!

**Pull request merged**